Userland/Libraries/LibCore/Stream.h

```diff
@@ -703,7 +703,7 @@ class BufferedHelper {
         Optional<size_t> longest_match;
         size_t match_size = 0;
         for (auto& candidate : candidates) {
-            auto const result = m_buffer.offset_of(candidate, readable_size);
+            auto const result = m_buffer.offset_of(candidate, {},readable_size);
```

Comment:
Missing space after `{},`, unchecked by the linter because it's removed in a future commit.